README.md

```diff
@@ -43,3 +43,37 @@ body:
 {
   "message": "Food not found."
 }
+```
+
+##### Update one food item
+
+A single food object can be updated by submitting a `PATCH` request to the `/api/v1/foods/:id` endpoint where `:id` is a valid Id of a food item in the system and the body contains both or one of name or calories as keys. This will return the updated Food Object with its attributes including its `id`, `name`, and `calories` along with a 200 status code. An example request would look like:
+
+```HTTP
+POST '/api/v1/foods/1'
+
+body:
+{
+  name: 'newname',
+  calories: 0
+}
+```
+The response would like:
```

Comment:
Might want to make this "The response would look like"

Reply:
Updated